### DISCUSSION TITLE
transport encryption required when enable LDAP /Active domain directory

### DISCUSSION THREAD

**Discussion by badrshs** [requarks/wiki#3363] [Help / Questions]
**Describe the bug**
when I login I got **transport encryption required** 

**To Reproduce**
1. enable 'LDAP /Active domain directory"
2. Go to 'Login as LDAP /Active domain directory'
3. Click on Login
4. See error  **transport encryption required** 

**Expected behavior**
I should be able to login as a user from my active domain

**Screenshots**
![image](https://user-images.githubusercontent.com/26596347/77829255-8cea8900-7129-11ea-8f72-6c1b75d1c81f.png)

**Host Info (please complete the following information):**
 - OS: Ubuntu 18.04
 - Wiki.js version: 2.1.113
 - Database engine: mysql 5.7

**Comment by NGPixel** [comment]
Did you enable TLS in the LDAP module settings?

**Comment by badrshs** [comment]
no, we don't have TLS  on our local server

**Comment by NGPixel** [comment]
The error says otherwise.
